compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/insert_aie_workgroup.mlir

```diff
@@ -0,0 +1,138 @@
+// RUN: iree-opt --split-input-file --pass-pipeline="builtin.module(iree-amdaie-insert-aie-workgroup)" %s | FileCheck %s
+
+// CHECK-LABEL: @insert_aie_workgroup
+// CHECK: scf.forall (%arg0, %arg1) in (1, 1) {
```

Comment:
Here and in other parts of the CHECK statements - we shouldn't use the SSA values directly as it is in the lit test.

`%[[VARIABLE:.*]] = ` for defining and `%[[VARIABLE]]` for using it is what we should do.

Reply:
Done